README.md

```diff
@@ -124,7 +124,7 @@ It can be started with the following command:
 
     docker run -d --name jupyter.cont [-v /home/jupyter-home:/home] jupyter/jupyterhub jupyterhub
 
-This command will create a container that you can stop and resume with `docker stop/start`.
+This command will create a named container, `jupyter.cont`, that you can stop and resume with `docker stop/start`.
```

Comment:
... a container named `jupyter.cont` that you...

but that is just nitpicking.